plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -103,6 +112,26 @@ public CachingJdbcClient(
             Duration metadataCachingTtl,
             boolean cacheMissing,
             long cacheMaximumSize)
+    {
+        this(delegate,
+                sessionPropertiesProviders,
+                identityMapping,
+                metadataCachingTtl,
+                Optional.empty(),
+                Optional.empty(),
+                cacheMissing,
+                cacheMaximumSize);
+    }
```

Comment:
Can we remove this constructor use the other one at all places ?

Reply:
Those options are now non-optional and will have to be equal to `metadataCachintTtl` in such cases. I think it's easier to have an overloaded constructor to indicate that.